packages/core/js-sdk/src/admin/product.ts

```diff
@@ -58,17 +58,93 @@ export class Product {
     )
   }
 
+  /**
+   * This method creates a product import. The products are only imported after
+   * the import is confirmed using the {@link confirmImport} method.
+   *
+   * This method sends a request to the
+   * [Create Product Import](https://docs.medusajs.com/api/admin#products_postproductsimport)
+   * API route.
+   *
+   * @param body - The import's details.
+   * @param query - Query parameters to pass to the request.
+   * @param headers - Headers to pass in the request.
+   * @returns The import's details.
+   *
+   * @example
+   * sdk.admin.product.directImport({
+   *   file // uploaded File instance
+   * })
+   * .then(({ transaction_id }) => {
+   *   console.log(transaction_id)
+   * })
+   */
+  async directImport(
```

Comment:
thought: I am not loving the naming here because, but I also don't have any great suggestions for alternatives 😅 

Maybe we can do `createImport`?

Reply:
Yeah, same. Struggled with naming here a bit. I think we can go with `createImport`, since there is another method called `confirmImport`, so they both feel a bit in sync